scripts/mods/mod_reforged/dynamic_spawns/units/bandit_units.nut

```diff
@@ -7,6 +7,18 @@ figure_bandit_04 - Bandit Leader
 
 // We add our new bandit units
 local units = [
+	{
+		ID = "Bandit.Thug",
```

Comment:
No, this is not the way to change figures of vanilla units. You should change them the same way you change their Cost i.e. `::DynamicSpawns.Units.findById("xyz").Figure = "abc"`.

The reason the current implementation doesn't work is because this is a NEW unit being registered with DSF. And it is missing the `Troop` key.

Reply:
Unfortunately I merged the PR first and then realized this mistake. So I deleted the merged commits and restored this branch. This is gonna cause GitHub issues whereby we can't merge this PR again. But that's ok, we will cherry pick the commits onto the development branch.

Reply:
Basically, I'd like that DSF no longer has its own unit/unitblocks/parties definitions and we port them all to Reforged. Then we won't run into this issue as all the units will be defined by us.